LaunchSpatial.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
+echo Starting Spatial with the default launch configuration. Modify LaunchSpatial.bat if you want to use a different launch configuration.
+
+pushd "%~dp0\spatial"
+spatial worker build build-config
+
+spatial local launch
```

Comment:
Let's actually pass `default_launch.json` here and add a `rem` comment to replace this with the desired launch configuration.